VisualC/pkg-support/cmake/sdl2-config.cmake

```diff
@@ -0,0 +1,77 @@
+# SDL2 CMake configuration file:
+# This file is meant to be placed in a cmake subfolder of SDL2-devel-2.x.y-VC
+
+cmake_minimum_required(VERSION 3.0)
+
+set(_sdl2_known_comps SDL2 SDL2main SDL2test)
```

Comment:
```suggestion
set(_sdl2_known_comps SDL2 SDL2main SDL2_test)
```

Reply:
I'm going to keep `SDL2test` as the COMPONENT name + name of target (`SDL2::SDL2test`).
See https://github.com/libsdl-org/SDL/issues/3519#issuecomment-1137700193
I feel this gives more natural for users.

The current cmake build script does not install SDL2 library, so I'm going to create/install the same target there.
(In CMake, there is no need to name the library (=`libSDL2_test.a`) and the target (=`SDL2::SDL2test`) the same.